### PR TITLE
Support responsive page

### PR DESCRIPTION
**Pull request by senyer**
image

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -18,34 +18,17 @@
...
-        <exclusion>
-          <groupId>com.distkv</groupId>
-          <artifactId>distkv-common</artifactId>
-        </exclusion>
-        <exclusion>
-          <groupId>ch.qos.logback</groupId>
-          <artifactId>logback-classic</artifactId>
-        </exclusion>
-      </exclusions>
+      <version>0.1.4-SNAPSHOT</version>
```
Comment (jovany-wang) [inline pom.xml:26]: Why change this version?

File: src/main/java/cloud/distkv/api/RunnerApi.java [hunk 2]
```diff
@@ -23,4 +32,20 @@ public String index() {
     return "run/run";
   }
 
+  /**
+   * Execute cmd.
+   *
+   * @param cmd cmd content
+   * @return result
+   */
+  @PostMapping("cmd")
```
Comment (jovany-wang) [inline src/main/java/cloud/distkv/api/RunnerApi.java]: command

File: src/main/java/cloud/distkv/api/RunnerApi.java [hunk 3]
```diff
@@ -23,4 +32,20 @@ public String index() {
     return "run/run";
   }
 
+  /**
+   * Execute cmd.
+   *
+   * @param cmd cmd content
```
Comment (jovany-wang) [inline src/main/java/cloud/distkv/api/RunnerApi.java]: The command to execute.

File: src/main/java/cloud/distkv/api/RunnerApi.java [hunk 4]
```diff
@@ -23,4 +32,20 @@ public String index() {
     return "run/run";
   }
 
+  /**
+   * Execute cmd.
```
Comment (jovany-wang) [inline src/main/java/cloud/distkv/api/RunnerApi.java]: Execute a command.

File: src/main/resources/application.yml [hunk 5]
```diff
@@ -1,7 +1,7 @@
 server:
   servlet:
     context-path: /
-  port: 8080
+  port: 9527
```
Comment (jovany-wang) [inline src/main/resources/application.yml]: 12333

File: src/main/resources/static/js/vue.min.js [hunk 6]
```diff
@@ -0,0 +1,6 @@
...
...ode(t)},createComment:function(t){return document.createComment(t)},insertBefore:function(t,e,n){t.insertBefore(e,n)},removeChild:function(t,e){t.removeChild(e)},appendChild:function(t,e){t.appendChild(e)},parentNode:function(t){return t.parentNode},nextSibling:function(t){return t.nextSibling},tagName:function(t){return t.tagName},setTextContent:function(t,e){t.textContent=e},setAttribute:function(t,e,n){t.setAttribute(e,n)}}),Xo={create:function(t,e){Re(e)},update:function(t,e){t.data.ref!==e.data.ref&&(Re(t,!0),Re(e))},destroy:function(t){Re(t,!0)}},ta=new io("",{},[]),ea=["create","activate","update","remove","destroy"],na={create:Ve,update:Ve,destroy:function(t){Ve(t,ta)}},ra=Object.create(null),ia=[Xo,na],oa={create:We,update:We},aa={create:Ze,update:Ze},sa=/[\w).+\-_$\]]/,ca="__r",ua="__c",la={create:wn,update:wn},fa={create:xn,update:xn},pa=v(function(t){var e={},n=/;(?![^(]*\))/g,r=/:(.+)/;return t.split(n).forEach(function(t){if(t){var n=t.split(r);n.length>1&&(e[n[0].trim()]=n[1].trim())}}),e}),da=/^--/,va=/\s*!important$/,ha=function(t,e,n){if(da.test(e))t.style.setProperty(e,n);else if(va.test(n))t.style.setProperty(e,n.replace(va,""),"important");else{var r=ya(e);if(A
```
Comment (jovany-wang) [inline src/main/resources/static/js/vue.min.js:6]: A NEW LINE

File: src/main/resources/static/js/axios.min.js [hunk 7]
```diff
@@ -0,0 +1,9 @@
...
+//# sourceMappingURL=axios.min.map
```
Comment (jovany-wang) [inline src/main/resources/static/js/axios.min.js:9]: new line

File: src/main/resources/static/js/vue.min.js (hunk 6 above)
Comment (senyer) [reply src/main/resources/static/js/vue.min.js:6]: This is an external file reference, we better not change it.

File: src/main/resources/static/js/axios.min.js (hunk 7 above)
Comment (senyer) [reply src/main/resources/static/js/axios.min.js:9]: This is an external file reference, we better not change it.

File: src/main/resources/application.yml (hunk 5 above)
Comment (senyer) [reply src/main/resources/application.yml]: OK

File: pom.xml (hunk 1 above)
Comment (senyer) [reply pom.xml:26]: Beacuse  0.1.3 will throws a exception: Class not found: org/dousi/ClientConfig.

**Review by jovany-wang — APPROVED** [review]

**Pull request merged**